config/environments/production.rb

```diff
@@ -25,8 +25,8 @@
 
   # Compress JavaScript.
   # Fix for ie8 and select2
-  require "uglifier"
-  config.assets.js_compressor = Uglifier.new(output: { ascii_only: true, quote_keys: true })
+  config.assets.js_compressor = :terser
+  config.assets.terser = { compress: { ascii_only: true, quote_keys: true } }
```

Comment:
Unsure if options still required, we dug out [this commit](https://github.com/argerim/select2-rails/pull/66/files).
Not supporting IE8 but the options are still valid options for Terser.

Reply:
Hmmm looking at the [terser-ruby README](https://github.com/ahorek/terser-ruby), this should be:

```suggestion
  config.assets.terser = { output: { ascii_only: true, quote_keys: true } }
```